pkg/controller/plan/builder/vsphere/builder.go

```diff
@@ -391,10 +436,35 @@ func (r *Builder) esxHost(vm *model.VM) (esxHost *EsxHost, found bool, err error
 	}
 	secret.Data["thumbprint"] = []byte(hostModel.Thumbprint)
 	esxHost = &EsxHost{
-		Inventory: r.Inventory,
+		Inventory: r.Source.Inventory,
 		Secret:    secret,
 		URL:       url,
 	}
 
 	return
 }
+
+//
+// Set volume and access modes.
+func (r *Builder) defaultModes(dm *mapped.DestinationStorage) (err error) {
+	model := &ocp.StorageClass{}
+	err = r.Destination.Inventory.Get(model, dm.StorageClass)
```

Comment:
This is the driver for the switch to the `Context`.  Currently, the `Builder` was only being passed the _source_ inventory client.  Here, we needed to find the storage class in the destination inventory.